src/pages/DevOps.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Fragment } from 'react';
+import { Button, Card, CardBody, Col, Row } from 'reactstrap';
+import ReactPlayer from 'react-player';
+import { useRouteData } from 'react-static';
+import { Link } from '@reach/router';
+import student1 from '../assets/images/photo1.jpg';
+import logophoto from '../assets/images/logodevops.jpg';
+
+const Dev = () => {
+  const { devops } = useRouteData();
+  return (
+    <div className="default-content">
+      <div className="content">
+
+        <h1>DevOps project</h1>
+      </div>
+      <Fragment>
+        <Row>
+          <Col xl="4" lg="6">
+            <Card className="mb-5">
+              <img alt="..." className="card-img-top" src={logophoto} />
```

Comment:
`...` is not sufficient alt text. you would either describe the image or say that the image is decorative `alt="Decorative Image"`